### PR TITLE
[Roslyn Analyzer - 2] Import Roslyn Analyzer using the environment variable

### PR DESCRIPTION
**Pull request by zhhyu**
Now we use the environment variable to import MSBuild project from NuGetBuild repo for Roslyn Analyzer and its configuration.

_Update $BuildBranchCommit before checking in_

### CODE REVIEW

File: src/NuGetCDNRedirect/NuGetCDNRedirect.csproj [hunk 1]
```diff
@@ -113,7 +113,7 @@
       <Version>1.1.3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.CodeDom.Providers.DotNetCompilerPlatform">
-      <Version>1.0.5</Version>
+      <Version>3.6.0</Version>
```
Comment (zhhyu) [inline src/NuGetCDNRedirect/NuGetCDNRedirect.csproj:116]: Upgrade the package to be compatible with the latest Roslyn Analyzer.

File: build.ps1 [hunk 2]
```diff
@@ -123,6 +123,9 @@ Invoke-BuildStep 'Restoring solution packages' { `
     -skip:$SkipRestore `
     -ev +BuildErrors
 
+Invoke-BuildStep 'Removing .editorconfig file' { Remove-EditorconfigFile -Directory $PSScriptRoot } `
```
Comment (zhhyu) [inline build.ps1:126]: See https://github.com/NuGet/ServerCommon/pull/377

**Review by dannyjdev — CHANGES_REQUESTED** [review]
Just make sure to update $BuildBranchCommit once https://github.com/NuGet/ServerCommon/pull/377 is merged.

File: build.ps1 [hunk 3]
```diff
@@ -9,7 +9,7 @@ param (
     [string]$SemanticVersion = '1.0.0-zlocal',
     [string]$Branch = 'zlocal',
     [string]$CommitSHA,
-    [string]$BuildBranchCommit = 'ade39b693d49b266ec5cac5d939edac7dda2fd92'
+    [string]$BuildBranchCommit = '4e43763650ef1d3d7eb532484cc7889ce8cdfd86'
```
Comment (agr) [inline build.ps1:12]: Make sure you update the hash once https://github.com/NuGet/ServerCommon/pull/377 is merged.

**Review by agr — APPROVED** [review]

**Review by dannyjdev — APPROVED** [review]

**Pull request merged**